compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       - ./testnet:/etc/nomos
     environment:
       - NET_NODE_KEY=${DOCKER_COMPOSE_BOOSTRAP_NET_NODE_KEY:-1000000000000000000000000000000000000000000000000000000000000000}
-    command: /etc/nomos/config.yaml
+    command: /etc/nomos/bootstrap_config.yaml
```

Comment:
we may need to add the following for the bootstrap? because the bootstrap node will try to connect to `mix-node-2:7778` when it starts up.
```
depends_on:
  - etcd
  - mix-node-0
  - mix-node-1
  - mix-node-2
```

Plus, we should merge https://github.com/logos-co/nomos-node/pull/445. Unless, the bootstrap node will be sometimes not able to connect to the `mix-node-2`.

Reply:
I'll rebase it on #455, if it makes any difference I'll merge this after #455 is in master.

Reply:
For testing in my local, I added a 3s sleep temporarily before connecting to `mix-node-2:7778`. The bootstrap node was connected with `mix-node-2` successfully. And, I saw that messages were broadcasted to all nomos-nodes (both bootstrap and node-1) succesfully.

But, even though the vote from node-1 (for the view 0) was delivered to the bootstrap node (the first leader), it didn't propose the next block. I'm taking a look at details. I guess that tally didn't meet but I don't know why yet.

Reply:
I think thats related to tree overlay and the next leader being offline. You did a fix in integration tests for that, I'm thinking of adding a command to nomos-cli that would return the topology, includong next leader, when a node config is provided. I'll do this in new PR.

Reply:
I think I found the reason why the tally for the view 0 is not met.

When we run the bootstrap node, we don't configure the `overlay.nodes`. We need to configure it as we do for subsequents nodes (by `configure_node.sh`). 

I observed that messages were delivered between two nodes successfully. But, the committee ID (that we use as a recipient when sending messages) were different between two nodes, because of the reason above.